bedrock/bedrock.py

```diff
@@ -284,6 +287,9 @@ def save(self, db, force=False):
         for block in palette:
           data += nbt.encode(block)
 
+      if self.version == 9:
+        data = self.y_db + data

```

Comment:
Have you tested this code? I don't think you can concatenate an integer and a bytes object.

Reply:
Yes, it works